From 7b565af4f41615bad43f710514a4f8fb9d2d2721 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Amadeusz=20=C5=BBo=C5=82nowski?= <aidecoe@aidecoe.name>
Date: Tue, 31 Jul 2012 10:49:58 +0200
Subject: [PATCH 3/5] 99shutdown: remove --no-wall argument for reboot, halt,
 etc...

In some distros (Gentoo, for example) these commands don't honor
--no-wall argument.
---
 modules.d/99shutdown/shutdown.sh |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/modules.d/99shutdown/shutdown.sh b/modules.d/99shutdown/shutdown.sh
index aa0a81f..e1da91b 100755
--- a/modules.d/99shutdown/shutdown.sh
+++ b/modules.d/99shutdown/shutdown.sh
@@ -56,7 +56,7 @@ done
 _check_shutdown final
 
 getarg 'rd.break=shutdown' && emergency_shell --shutdown shutdown "Break before shutdown"
-[ "$1" = "reboot" ] && reboot -f -d -n --no-wall
-[ "$1" = "poweroff" ] && poweroff -f -d -n --no-wall
-[ "$1" = "halt" ] && halt -f -d -n --no-wall
+[ "$1" = "reboot" ] && reboot -f -d -n
+[ "$1" = "poweroff" ] && poweroff -f -d -n
+[ "$1" = "halt" ] && halt -f -d -n
 [ "$1" = "kexec" ] && kexec -e
-- 
1.7.8.6

